### PR TITLE
fix: Add Q/A for security bounty program

### PR DESCRIPTION
**Pull request by spark-c**
Adds a FAQ/A to the [FAQ page](https://docs.appsmith.com/getting-started/faq) regarding Appsmith's policy for rewarding security reporting.

Notion [source doc here](https://www.notion.so/appsmith/When-Users-Want-to-Report-Security-Vulnerabilities-8d6938c33fc44fb7a9f2e179dc65f676)

The linter will be angry, but since this page is much more conversational and isn't tech docs, what rules should we follow? Generally, the rules that this page breaks are:

- using the `we/our` 1st person pronouns
- `?` punctuation in the subheadings
- use of `!` for happiness and enthusiasm

### CODE REVIEW

**Comment by jnikhila** [comment]
> Adds a FAQ/A to the [FAQ page](https://docs.appsmith.com/getting-started/faq) regarding Appsmith's policy for rewarding security reporting.
> 
> Notion [source doc here](https://www.notion.so/appsmith/When-Users-Want-to-Report-Security-Vulnerabilities-8d6938c33fc44fb7a9f2e179dc65f676)
> 
> The linter will be angry, but since this page is much more conversational and isn't tech docs, what rules should we follow? Generally, the rules that this page breaks are:
> 
> * using the `we/our` 1st person pronouns
> * `?` punctuation in the subheadings
> * use of `!` for happiness and enthusiasm

@spark-c - I agree - we can ignore the linter issues for the FAQ page.

File: website/docs/getting-started/faq.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Yes, absolutely. You can write JS code anywhere inside the product within `{{ }}
 
 Appsmith safely encrypts all your database credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Appsmith is an open-source platform and can be fully audited and deployed on-premise to ensure none of your data leaves your VPC.
 
+## Do you have a bounty program for reporting security vulnerabilities?
```
Comment (jnikhila) [inline website/docs/getting-started/faq.md:20]: I think - This question should be added as below:

Q: Do you have a bounty program for reporting security vulnerabilities?
A: No, we do not currently have a bounty program for reporting security vulnerabilities. However, we are always grateful for any information that can help us improve the security of our systems and protect our users' data.

Q: How do I report a security vulnerability?
A: If you believe you have discovered a security vulnerability, please email our security team at security@appsmith.com with a description of the issue and any relevant details. We will review your report and take appropriate action to address the issue.

Comment (danciaclara) [reply website/docs/getting-started/faq.md:20]: Agree with @jnikhila on this. Just a small modification to the content

Q: How do I report a security vulnerability?
A: We appreciate any information that can help us improve the security of our systems and protect our users' data. If you believe you have discovered a security vulnerability, please email our security team at [security@appsmith.com](mailto:security@appsmith.com) with a description of the issue and any relevant details. We will review your report and take appropriate action to address the issue.

File: website/docs/getting-started/faq.md [hunk 2]
```diff
@@ -7,36 +7,42 @@ sidebar_position: 4
...
 
 ## Can I write code in Appsmith?
 
-Yes, absolutely. You can write JS code anywhere inside the product within `{{ }}`. We know how complex and intricate business logic can be and trying to build it without code can get excruciating. Appsmith bridges mundane API and UI configurations with powerful code expressions to let you build the apps the way you see them fit.
+Yes, absolutely. You can write JavaScript code anywhere inside the product within curly braces `{{ }}`, or using standard syntax within [JS Objects](/core-concepts/writing-code/javascript-editor-beta). We know how complex and intricate business logic can be, and trying to build it without code can get excruciating. Appsmith bridges mundane API and UI configurations with powerful code expressions to let you build the apps the way you see them fit.
 
-## What type of data security does appsmith provide?
+Read more about writing code [here](/core-concepts/writing-code).
```
Comment (jnikhila) [inline website/docs/getting-started/faq.md:16]: Could you please follow the style guide for links? (https://developers.google.com/style/link-text) and make the necessary changes.

**Review by jnikhila — CHANGES_REQUESTED** [review]

**Comment by spark-c** [comment]
@jnikhila @danciaclara I think this should be good to go now! Will merge once Nikhila's review/requested changes are cleared.

**Pull request merged**